package.json

```diff
@@ -44,9 +45,10 @@
   "homepage": "https://github.com/czue/bluesky-comments#readme",
   "exports": {
     ".": {
-      "import": "./dist/bluesky-comments.es.js",
+      "import": "./src/main.jsx",
```

Comment:
did you change this on purpose? I'm not a packaging expert, but I think I set this to get something working properly on one of the JS CDNs.

Reply:
I'm a packaging idiot, so was stumbling along tweaking things to get imports working